HACKING.md

```diff
@@ -65,3 +65,18 @@ To test the default `ClusterIngress` manifest:
 ```
 $ oc create -f deploy/cr.yaml
 ```
+
+## Tests
+
+### Integration tests
+
+Integration tests are still very immature. To run them, use the GCP test cluster instructions, and then run the tests using the following as a guide:
+
+```
+KUBERNETES_CONFIG=/some/kubeconfig \
```

Comment:
do you really mean KUBERNETES_CONFIG or KUBECONFIG?

Reply:
No, this is a weird peculiarity of the operator-sdk which uses KUBERNETES_CONFIG for some reason...